appfl/algorithm/iiadmm.py

```diff
@@ -141,13 +141,14 @@ def update(self):
             super(IIADMMClient, self).laplace_mechanism_output_perturb()
 
         """ Update local_state """
-        self.local_state = OrderedDict()                
-        self.local_state["primal"] = copy.deepcopy(self.primal_state)                        
+        self.local_state = OrderedDict()
+        self.local_state["primal"] = copy.deepcopy(self.primal_state)
+        self.local_state["dual"] = OrderedDict()
```

Comment:
@minseok-ryu please confirm that this change is ok.

Reply:
It is okay!